docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -139,48 +139,49 @@ The panel's **Attribute Guide** section displays the available attributes for th
+![The Viam app showing the detectionCam component section. The Attributes section contains a skeleton configuration, including source, pipeline, type, and attributes. The Attributes Guide section lists the available camera component attributes. There are buttons labeled Data Capture Configuration, and Frame, and a drop-down labeled, Depends On. On the upper right there is a trash bin icon.](/tutorials/try-viam-color-detection/transform-camera-config.png)
 
-Copy the following JSON configuration into the Attributes section:
+Copy the following JSON configuration into the **Attributes** section:
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-  "source": "cam",
-  "pipeline": [
-    {
-      "attributes": {
-        "detector_name": "my_color_detector",
-        "confidence_threshold": 0.3
-      },
-      "type": "detections"
-    }
-  ]
+ "source": "cam",
+ "pipeline": [
+   {
+     "attributes": {
+       "detector_name": "my_color_detector",
+       "confidence_threshold": 0.3
+     },
+     "type": "detections"
+   }
+ ]
 }
 ```
 
-<!-- prettier-ignore -->
 | Field | Default value | Description |
 | ----- | ------------- | ----------- |
 | `source` | `cam` | The name of the physical camera on the rover, which provides the visual feed to get detections from. |
 | `attributes` | | The attributes of this detectionCam. |
 | `attributes.detector_name` | `my_color_detector` | The name of this detectionCam. |
-| `attributes.confidence_threshold` | `0.3` (30%) | The percentage of confidence in the color being present the detection service needs to detect a color. |
+| `attributes.confidence_threshold` | `0.3` (30%) | The percentage of confidence needed by the detection service to identify a color. |
 | `type` | `detections` | The type of the component. |
 
-![The Viam app showing the detectionCam component section. It contains the Attributes section with a skeleton configuration, including source, pipeline, type, and attributes. The panel has an Attributes section populated with transform camera component attributes. The are buttons labeled Data Capture Configuration, and Frame, and a drop-down labeled, Depends On. On the upper right there is a trash bin icon.](/tutorials/try-viam-color-detection/detectioncam-component-panel.png)
+![The Viam app showing the detectionCam component section. The Attributes section contains a skeleton configuration, including source, pipeline, type, and attributes. The Attributes Guide section lists the available camera component attributes. There are buttons labeled Data Capture Configuration, and Frame, and a drop-down labeled, Depends On. On the upper right there is a trash bin icon.](/tutorials/try-viam-color-detection/transform-camera-attributes.png)
 
 After adding the component and its attributes, click **Save config**.
 
-## Test your transform camera in the CONTROL tab
+## Test your transform camera in the **Control** tab
```

Comment:
```suggestion
## Test your transform camera in the Control tab
```
[nit] not necessary to bold when it's part of a heading. Good catch though and you're right to de-all-caps.